static/src/javascripts/projects/common/modules/navigation/editionalise-menu.js

```diff
@@ -36,5 +36,5 @@ define([
         }
     }
 
-    return editionaliseMenu();
```

Comment:
I realise this doesn't make sense to invoke here, its already being invoked in another file 😅